railties/test/generators/app_generator_test.rb

```diff
@@ -861,7 +861,12 @@ def test_edge_option
   end
 
   def test_master_option
```

Comment:
```suggestion
  def test_main_option
```

Reply:
that test is defined separately https://github.com/rails/rails/pull/41141/files#diff-9dd1c2e31fd5617506e344d6328e0e165614290478e1180ef0630017b9305ba4R868

Reply:
Ah, good point.